grc/gui_qt/components/wiki_tab.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2014-2022 Free Software Foundation, Inc.
```

Comment:
We don't have to assign to the FSF – the copyright can remain with you, @jholtom if you want it.